feature-group-definitions/cap.dist.yml

```diff
@@ -0,0 +1,18 @@
+# Generated from: cap.yml
+# Do not edit this file by hand. Edit the source file instead!
+# Last modified: 2024-04-04T09:35:23.998323918+00:00[UTC]
+
+name: cap unit
+description: The CSS `cap` unit corresponds to the height of Latin capital letters.
+spec: https://drafts.csswg.org/css-values-4/#cap
+status:
+  baseline: false
+  support:
+    chrome: "118"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/length#browser_compatibility says 117

Reply:
`api.CSS.cap_static` [says 118](https://developer.mozilla.org/en-US/docs/Web/API/CSS#browser_compatibility) and that seems to be the truth of it: https://github.com/mdn/browser-compat-data/pull/20984#pullrequestreview-1724941202

Reply:
I guess that's a small enough difference that we just go with it, even though web developers will use `cap` as a unit much more often than `CSS.cap()`.